packages/outputarea/src/widget.ts

```diff
@@ -1555,20 +1555,17 @@ namespace Private {
       onClick: (event: MouseEvent) => void
     ) {
       const node = document.createElement('div');
-      const title = `The first ${maxNumberOutputs} are displayed`;
-      const msg = 'Show more outputs';
-      node.insertAdjacentHTML(
-        'afterbegin',
-        `<a title=${title}>
-          <pre>${msg}</pre>
-        </a>`
-      );
+      const button = document.createElement('button');
+      button.type = 'button';
+      button.className = 'jp-TrimmedOutputs-button';
+      button.title = `The first ${maxNumberOutputs} are displayed`;
+      button.textContent = 'Show more outputs';
```

Comment:
While at it, would you mind adding the translation wrappers for the strings? It's interesting how these were missed as we have nearly 100% translation coverage.

Reply:
Done in 798ffeae2af570736040a55ab15570015eea469f